android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormBaseActivity.java

```diff
@@ -90,7 +91,10 @@ protected void onCreate(Bundle savedInstanceState) {
         onActivityRequestPermissionResultListeners = new HashMap<>();
         lifeCycleListeners = new ArrayList<>();
         isFormFragmentInitialized = false;
-        translateForm = getIntent().getBooleanExtra(JsonFormConstants.PERFORM_FORM_TRANSLATION, false);
+        translateForm = getIntent().hasExtra(JsonFormConstants.PERFORM_FORM_TRANSLATION) ?
+                getIntent().getBooleanExtra(JsonFormConstants.PERFORM_FORM_TRANSLATION, false) :
+                NativeFormLibrary.getInstance().isPerformFormsTranslation();
```

Comment:
This can be simplified to `getIntent().getBooleanExtra(JsonFormConstants.PERFORM_FORM_TRANSLATION, false) || NativeFormLibrary.getInstance().isPerformFormsTranslation();`

Reply:
Unless the idea is to allow `JsonFormConstants.PERFORM_FORM_TRANSLATION` to override the global config per form.

Reply:
Yes exactly, I plan to override the global config-if any single form needs to be un-translated.